ocs_ci/ocs/node.py

```diff
@@ -1881,3 +1881,20 @@ def add_new_disk_for_vsphere(sc_name):
     ]
     node_with_min_pvs = min(num_of_pv_per_node_tuples, key=itemgetter(0))[1]
     add_disk_to_node(node_with_min_pvs)
+
+
+def get_odf_zone_count():
+    """
+    Get the number of Availability zones used by ODF cluster
+
+    Returns:
+         int : the number of availability zones
+    """
+    node_obj = OCP(kind="node")
+    az_count = node_obj.get(selector=constants.ZONE_LABEL)
+    az = set()
+    for node in az_count.get("items"):
+        node_lables = node.get("metadata").get("labels")
```

Comment:
Just a note, we can also use the get code like below which will reduce using multiple '.get' again and again.
`node.get("metadata")["labels"]`
reference code: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/machine.py#L566

Reply:
fixed.